clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -282,3 +291,17 @@ func getLatencyThreshold(call *apiCall) time.Duration {
 	}
 	return latencyThreshold
 }
+
+type apiResponsivenessPrometheus struct {
+	metric *apiResponsiveness
+}
+
+// SummaryName returns name of the summary.
+func (*apiResponsivenessPrometheus) SummaryName() string {
+	return apiResponsivenessPrometheusMeasurementName
+}
+
```

Comment:
SummaryTime() is needed:
https://github.com/kubernetes/perf-tests/pull/490

Reply:
Added.